mpi-proxy-split/mpi-wrappers/mpi_request_wrappers.cpp

```diff
@@ -168,6 +173,9 @@ USER_DEFINED_WRAPPER(int, Testany, (int) count,
       break;
     }
   }
```

Comment:
Delete the next three lines:  `if (all_requests_null) { ... }`.

If `MPI_Test` was previously called and failed, then `MPI_Test` will have already set `*local_flag = 0;`.
If `MPI_Test` was never called, then `*local_flag` still has its initial value of `1`, which is the correct value when `all_requests_are_null` is true.